.github/workflows/test-indicators-coverage.yml

```diff
@@ -10,7 +10,7 @@ permissions:
 jobs:
   test:
     name: code coverage
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
Not an issue, but curious, does this simply run better on MacOS or do you see it as the most problematic (conservative) choice?

Reply:
I chose macos just for easily changing `locale` setting. When I tested on ubuntu, `locale` is not changed somehow🤔. If there's a reason we should keep it as ubuntu, please let me know.

Reply:
And also changed locale setting was not kept between steps. That's why the locale changing is run before pytest command in a single step.

Reply:
no reason, just curious.  I think for Ubuntu and Windows, you have to set an actual `LANG` environment variable or something like that.